frontend/src/components/create-user-page/CreateUser.scss

```diff
@@ -0,0 +1,44 @@
+@import "../../constants.scss";
+
+.create-user-form-heading {
+  text-align: center;
+}
+
+.create-user-form-field {
+  min-height: 55px;
+  width: 300px;
+}
+
+.create-user-form-field__label {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.create-user-form-field__input {
+  width: 60%;
+  padding: 5px;
+}
+
+.create-user-form-message {
+  text-align: right;
+  font-size: 12px;
+  color: red;
+  margin: 2px;
+}
```

Comment:
As above, nest this SCSS inside the `.create-user-form__field`

Reply:
done, thanks
I removed the className and have identified it by tag inside the .create-user-form__field className as the p tag coincided with the create-user-form-message className